_data/sites.json

```
@@ -7210,6 +7221,17 @@
         ]
     },
 
+    {
+        "name": "NAVER",
+        "url": "https://nid.naver.com/user2/help/myInfo",
+        "difficulty": "medium",
+        "notes": "You can delete it by clicking 'Delete account and data' at the bottom left of the My Info page. Note: Re-registration is not possible with a deleted ID, and posts written on the bulletin board are not automatically deleted. So, make it private or delete it before you delete it. ",
```

```suggestion
        "notes": "You can delete it by clicking 'Delete account and data' at the bottom left of the My Info page. Note: Re-registration is not possible with a deleted ID, and posts written on the bulletin board are not automatically deleted. So, make delete or make them private before you terminate your account. ",
```
Minor wording suggestion to avoid repetition.

---

This review was reflected in commit [0c34595](https://github.com/jdm-contrib/jdm/pull/969/commits/0c34595d30660fef3bd905a0658c282d5f4bb9d4) along with the review below.